utils/vara-ui/src/components/modal/modal.tsx

```diff
@@ -60,7 +60,9 @@ const Modal = ({ heading, close, children, className, footer }: Props) => {
         <header className={styles.header}>
           <h3 className={styles.heading}>{heading}</h3>
 
-          <Button icon={CrossSVG} color="transparent" onClick={close} className={styles.button} />
```

Comment:
What's the point of replacing library's `Button`? That's basically styles duplication.